File: Python/vowels.py
sentence = "The quick brown fox jumps over the lazy dog"
VOWELS = "AEIOUYaeiouy"

# Initiate counter
count = 0

# Don't need an array here, so we loop through sentence directly
for letter in sentence:
    # Look for this letter in the vowels string
    # If the letter is found in the VOWELS, this method returns something other than -1
    if(VOWELS.find(letter) != -1):
        count = count + 1

# Print the result
print(count)

File: Julia/vowels.jl
sentence = "The quick brown fox jumps over the lazy dog"
VOWELS = "AEIOUYaeiouy"

# Count the number of variables
counter  = 0

# Loop through the string directly
for i in 1:length(sentence)
    if occursin(sentence[i], VOWELS)
        # Refer to the global variable "counter"
       global counter = counter + 1
    end
end

println(counter)
